src/Components/Admin/EnvironmentEditor/environmentBar.js

```diff
@@ -111,9 +159,11 @@ export default function EnvironmentBar({
                         </Menu>
                     </div>
                     <div className={classes.buttonWrapperRight}>
-                        <Button className={classes.button}> Preview </Button>
                         <Button className={classes.button}>
-                            Share and Publish
+                            Share & Publish
+                        </Button>
```

Comment:
If you look at the behaviour of buttons on your Loom, it will change to the background colour when hovered on. Instead, can you change the hover action to go to a darker shade of the same button colour?